test/decl/class/override.swift

```diff
@@ -160,6 +160,26 @@ class H : G {
   override init(b: Double) {} // expected-error{{initializer does not override a designated initializer from its superclass}} {{none}}
 }
 
+class J {
+  func f1(_: Int, int1: Int) { } // expected-note{{potential overridden instance method 'f1(_:int1:)' here}} {{none}}
```

Comment:
For what I understood the intention of this PR is to add a new fix-it  but all tests fix-its are `{{none}}`(meaning this would not produce a fix-it on this line) ... not sure if  is that what it was intended

Reply:
Fixed it